pageStyles/constants.scss

```diff
@@ -11,3 +11,13 @@ $mobileSubHeaderFontSize: 20px;
 $contentFontSize: 20px;
 $buttonFontSize: 15px;
 
+//List Style
+@mixin sharedListStyle {
+  list-style: none;
+  display: block;
+}
+
+@mixin sharedLinkShare {
+  text-decoration: none;
+}
```

Comment:
I don't think we need the words `shared` or `style` here - because that's kind of already implied because its a sass mixin.

Reply:
renamed